src/index.js

```diff
@@ -28,7 +28,17 @@ export default function (sails) {
       };
 
       // Console Transport
-      logger = new winston.Logger({transports: [new winston.transports.Console(consoleOptions)]});
+      logger = new winston.Logger({
+        transports: [new winston.transports.Console(consoleOptions)],
+        levels: {
+          error: 1,
+          warn: 2,
+          debug: 3,
+          info: 4,
+          verbose: 5,
+          silly: 6
+        }
```

Comment:
These levels are taken from the Sails docs [here](http://sailsjs.org/documentation/concepts/logging).

Ideally this wouldn't be hard-coded, because now it won't update if sails adds a log level in the future. However, sails doesn't expose its internal log priority as far as I can tell, so hard-coding it appears to be the only option.